numbers.qmd

```diff
@@ -293,12 +293,12 @@ ggplot(money, aes(year, money)) +
   scale_y_log10()
 ```
 
-This a straight line because a little algebra reveals that `log(money) = log(starting) + n * log(interest)`, which matches the pattern for a line, `y = m * x + b`.
+This a straight line because a little algebra reveals that `log10(money) = log10(interest) * year + log10(starting)`, which matches the pattern for a line, `y = m * x + b`.
```

Comment:
To keep it as clear as possible I would add log10 here as well as swap the parts of the formula so that it actually matches the pattern of a line, i.e. y = mx +b.